vmhost/vmhooks/smallIntOps.go

```diff
@@ -244,11 +244,15 @@ func v1_4_smallIntStorageLoadUnsigned(context unsafe.Pointer, keyOffset int32, k
 		return 0
 	}
 
-	data, usedCache, err := storage.GetStorage(key)
+	data, trieDepth, usedCache, err := storage.GetStorage(key)
 	if vmhost.WithFault(err, context, runtime.BaseOpsErrorShouldFailExecution()) {
 		return 0
 	}
-	storage.UseGasForStorageLoad(smallIntStorageLoadUnsignedName, metering.GasSchedule().BaseOpsAPICost.Int64StorageLoad, usedCache)
+	err = storage.UseGasForStorageLoad(smallIntStorageLoadUnsignedName, int64(trieDepth), metering.GasSchedule().BaseOpsAPICost.Int64StorageLoad, usedCache)
+	if err != nil {
```

Comment:
please refactor this to resemble what we have in v1.5:
```GO
err = storage.UseGasForStorageLoad(smallIntStorageLoadUnsignedName, trieDepth, metering.GasSchedule().BaseOpsAPICost.Int64StorageLoad, usedCache)
if context.WithFault(err, runtime.BaseOpsErrorShouldFailExecution()) {
    return -1
}
```

Reply:
Done.